src/corehost/cli/args.h

```diff
@@ -42,6 +42,14 @@ struct probe_config_t
     {
     }
 
+    bool is_lookup() const
```

Comment:
These are the conditions that are set when nuget layout probe_path is set
look in https://github.com/dotnet/core-setup/blob/bee5e1f9718192b495016fb99097831943434984/src/corehost/cli/deps_resolver.cpp#L214